### PR TITLE
UICIRC-615: UI tests replacement with RTL/Jest for component RequestM…

### PR DESCRIPTION
**Pull request by pavelspichonak**
## Purpose
UI tests replacement with RTL/Jest for component RequestManagementSection

## Refs
https://issues.folio.org/browse/UICIRC-615

## Screenshots
Screen Shot 2021-09-28 at 1 42 24 PM

### CODE REVIEW

File: src/settings/LoanPolicy/components/EditSections/RequestManagementSection/RequestManagementSection.test.js [hunk 1]
```diff
@@ -0,0 +1,456 @@
...
+            loanable: true,
+          }}
+        />
+      );
+
+      expect(Period).toHaveBeenNthCalledWith(
+        periodCallOrderByPlace.minimumGuaranteedLoan,
+        {
+          fieldLabel: labelIds.minimumGuaranteedLoanPeriod,
+          inputValuePath: 'requestManagement.recalls.minimumGuaranteedLoanPeriod.duration',
```
Comment (Stanislau-Snapok) [inline src/settings/LoanPolicy/components/EditSections/RequestManagementSection/RequestManagementSection.test.js:225]: i think this one and next three props can be moced into separate constant value. you use them at least four times.

File: src/settings/LoanPolicy/components/EditSections/RequestManagementSection/RequestManagementSection.test.js [hunk 2]
```diff
@@ -0,0 +1,456 @@
...
+          type: 'checkbox',
+        }, {}
+      );
+    });
+  });
+
+  describe('"renew items with request" field', () => {
+    it('should render field', () => {
+      render(
+        <RequestManagementSection
```
Comment (Stanislau-Snapok) [inline src/settings/LoanPolicy/components/EditSections/RequestManagementSection/RequestManagementSection.test.js:436]: if i dont mistake, you render `RequestManagementSection` with this set of props at least five times. so can we put the cases with same render in one additional `describe` and use `beforeEach` construction? what do you think about it?

**Review by Stanislau-Snapok — APPROVED** [review]

File: src/settings/LoanPolicy/components/EditSections/RequestManagementSection/RequestManagementSection.test.js (hunk 1 above)
Comment (Dmitriy-Litvinenko) [reply src/settings/LoanPolicy/components/EditSections/RequestManagementSection/RequestManagementSection.test.js:225]: Could you please move 
```
intervalPeriods: intervarPeriodsExpectedValue,
changeFormValue: changeMock,
```
to variable and reuse it across all file?

Comment (pavelspichonak) [reply src/settings/LoanPolicy/components/EditSections/RequestManagementSection/RequestManagementSection.test.js:235]: done

File: src/settings/LoanPolicy/components/EditSections/RequestManagementSection/RequestManagementSection.test.js (hunk 2 above)
Comment (pavelspichonak) [reply src/settings/LoanPolicy/components/EditSections/RequestManagementSection/RequestManagementSection.test.js:440]: I already have such describe `when "policy.loanable" is true` but to be honest I would not like to move these tests there. The idea is to test all components separately. We can discuss it.

Comment (pavelspichonak) [reply src/settings/LoanPolicy/components/EditSections/RequestManagementSection/RequestManagementSection.test.js]: done

**Pull request merged**